pom.xml

```diff
@@ -112,19 +112,19 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.3.2</version>
+            <version>2.16.1</version>
```

Comment:
Why this version? Which CVEs are the older versions suffering from?

Reply:
For example https://nvd.nist.gov/vuln/detail/CVE-2023-35116 is affecting jackson-databind up to 2.15.2. Also the list of CVEs affecting 2.14.x and 2.15.x contains 13 entries (https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&orderBy=CPEURI&keyword=jackson-databind&status=FINAL&startIndex=160), so I think that an update of jackson-databind on all consumers of the model-jacksonexporter is warranted.

Reply:
https://nvd.nist.gov/vuln/detail/CVE-2023-35116 is questionable in this context (compare with https://github.com/FasterXML/jackson-databind/issues/3972), also that would only require 2.16.0 (not 2.16.1), the other ones only apply to much older versions. According to https://security.snyk.io/package/maven/com.fasterxml.jackson.core:jackson-databind the version 2.13.4.1 (from 2022) or newer should be pretty safe.

Reply:
I am fine with that. In that case I would like to set the version for all jackson dependency to 2.13.5